changelogs/current.yaml

```diff
@@ -66,23 +67,23 @@ minor_behavior_changes:
     :ref:`dns_cache_config <envoy_v3_api_field_extensions.filters.http.dynamic_forward_proxy.v3.FilterConfig.dns_cache_config>`.
 - area: grpc-json-transcoding
   change: |
-    Add SSE style message framing for streamed responses in :ref:`gRPC JSON transcoder filter <config_http_filters_grpc_json_transcoder>`.
+    Added SSE style message framing for streamed responses in :ref:`gRPC JSON transcoder filter <config_http_filters_grpc_json_transcoder>`.
 - area: http
   change: |
-   :ref:`response_headers_to_add <envoy_v3_api_field_config.route.v3.Route.response_headers_to_add>` and
-   :ref:`response_headers_to_remove <envoy_v3_api_field_config.route.v3.Route.response_headers_to_remove>`
-   will also be applied to the local responses from the ``envoy.filters.http.router`` filter.
+    :ref:`response_headers_to_add <envoy_v3_api_field_config.route.v3.Route.response_headers_to_add>` and
+    :ref:`response_headers_to_remove <envoy_v3_api_field_config.route.v3.Route.response_headers_to_remove>`
+    will also be applied to the local responses from the ``envoy.filters.http.router`` filter.
 - area: tracing
   change: |
-    Add :ref:`max_cache_size <envoy_v3_api_field_config.trace.v3.OpenTelemetryConfig.max_cache_size>`
+    Added :ref:`max_cache_size <envoy_v3_api_field_config.trace.v3.OpenTelemetryConfig.max_cache_size>`
     to the OpenTelemetry tracer config. This limits the number of spans that can be cached before flushing.
 - area: aws
   change: |
     :ref:`AwsCredentialProvider <envoy_v3_api_msg_extensions.common.aws.v3.AwsCredentialProvider>` now supports all defined credential
-    providers, allowing complete customisation of the credential provider chain when using AWS request signing extension.
+    providers, allowing complete customization of the credential provider chain when using AWS request signing extension.
```

Comment:
this one is arguably correct - altho perhaps we should standardize on us english